samples/ChangeFeedManagement/Program.py

```diff
@@ -127,7 +127,7 @@ def run_sample():
             ChangeFeedManagement.ReadFeedForTime(client, time)
 
         except errors.HTTPFailure as e:
-            print('\nrun_sample has caught an error. {0}'.format(e.message))
+            print('\nrun_sample has caught an error. {0}'.format(e))
```

Comment:
`message` is not an attribute on Exceptions in Python 3 like it is in Python 2.  `str(e)` is the same as `e.message` in Python 2 and works in Python 3.